Manager/src/main/java/io/raspberrywallet/manager/modules/Module.java

```diff
@@ -81,7 +82,7 @@ public String getStatusString() {
      *
      * @param status - new status
      */
-    void setStatusString(@NotNull String status) {
+    protected void setStatusString(@NotNull String status) {
```

Comment:
this methond will be used by manager to change the module status. So public modifier is required

Reply:
will change